x/datadeal/keeper/deal.go

```diff
@@ -603,7 +609,36 @@ func (k Keeper) DeactivateDeal(ctx sdk.Context, dealID uint64) error {
 		return sdkerrors.Wrapf(types.ErrDealDeactivate, err.Error())
 	}
 
-	//Todo:Remove DataVerification/DeliveryVote Queue
+	return nil
+}
+
+func (k Keeper) RemoveVotingQueue(ctx sdk.Context, DeactivateDealID uint64) error {
```

Comment:
I think it would be good not to remove the voting queue. Because this code will run very slowly if the  voting queue contains a lot of data.(searched all data).

Umm. How about ignoring deactive deal data as an endblocker.

Reply:
I think the method you suggested is better. It would be better to check the deal status in the verfication/delivery endblocker and ignore it when it is in the deactivate related state.

Reply:
So, I think this PR can be closed.
What do you think?

Reply:
Okay, sure. I closed this PR.